src/govuk/components/file-upload/_index.scss

```
@@ -11,6 +11,16 @@
     padding-top: $component-padding;
     padding-bottom: $component-padding;
 
+    // The default file upload button in Safari does not
+    // support setting a custom font-size. Set `-webkit-appearance`
+    // to `button` to drop out of the native appearance so the
+    // font-size is set to 19px
+    // https://bugs.webkit.org/show_bug.cgi?id=224746
+    &::-webkit-file-upload-button {
+      -webkit-appearance: button;
+      color: $govuk-text-colour;
```

Would inheriting from the parent work here? Just thinking that'd mean less things to keep in sync, and the print styles in the `govuk-text-colour` also get applied correctly.

```suggestion
      color: inherit;
```